pkg/storage/replica.go

```diff
@@ -905,6 +905,10 @@ func (r *Replica) updateProposalQuotaRaftMuLocked(
 	// ourselves via looking at MsgAppResp messages would be overkill. Perhaps
 	// another accessor on RawNode.
 	status := r.raftStatusRLocked()
+	if status == nil {
```

Comment:
should this be higher up? Or, in case we don't have a Raft group, what happens in the code above this?